packages/guideline-blocks-settings/src/components/RichTextEditor/RichTextEditor.spec.ct.tsx

```diff
@@ -82,7 +82,7 @@ describe('RichTextEditor', () => {
         cy.get(RteHtmlSelector).should('not.exist');
     });
 
-    it('should be able to select internal link', () => {
+    it('select internal link', () => {
```

Comment:
We should move these tests to vitests

Reply:
I get that and it makes sense in certain scenarios, just keep in mind that this will heavily increase the scope of this bugfix ticket as RTE tests are on the more complicated side. From a planning perspective I'm not sure this is very ideal.

In any way I can move the tests but that means I will need to shift some focus and will later get to my other tickets. I will leave the prioritisation to your judgement.

Reply:
Ok lets skip it this time ;)